_pages/dev/performance/performance-best-practices.md

```diff
@@ -41,6 +41,34 @@ You can improve the performance of your storefront application with the followin
 - Ensure you have an appropriate browser cache policy for all of the assets of the page. Browser caching saves a lot of bandwidth and improves page load time (both initial and subsequent) when it is properly set and tuned.
 
+## Image Format Recommendations
+
+You can improve the performance of your storefront by following these image format recommendations:
+
+- Using WebP or AVIF (AV1 Image File Format) can drastically reduce the size of your images, which can improve the performance of your storefront application.
+  - Check browser support for WebP and AVIF formats before using them.
+  - Provide fallback options for browsers that do not support WebP or AVIF formats.
+- Select the correct compression level for your images to balance image quality and file size.
+  - Use lossy compression for photographs and images with gradients.
+  - Use lossless compression for images with text, sharp edges, or when quality cannot be compromised.
+- Use CDN services that support WebP and AVIF formats to deliver images to users.
+- Use vector graphics (SVG) for simple images and icons, which can be scaled without losing quality and have smaller file sizes compared to raster images.
+- Use CSS techniques to create simple shapes and icons instead of using images. This can reduce the number of image files needed and improve loading times.
+
+## CDN Recommendations
+
+Using a CDN can significantly improve the performance of your storefront by distributing your content across multiple servers located around the world. Here are some best practices for using a CDN:
+
+- Choose a Reliable CDN Provider: Select a CDN provider that offers good performance, reliability, and coverage in the regions where your users are located.
+- Cache Static Assets: Configure your CDN to cache static assets such as images, CSS, JavaScript files, and fonts. This reduces the load on your origin server and speeds up content delivery to users.
+- Use HTTP/2: Make sure your CDN supports HTTP/2, which can improve the performance of your storefront by reducing latency and improving the efficiency of data transfer.
+- Enable Brotli Compression: Configure your CDN to compress content using Brotli compression to reduce the size of files transferred over the network.
+- Use a Secure Connection: Ensure that your CDN supports HTTPS to encrypt data in transit and protect user privacy.
+- Monitor Performance: Monitor the performance of your CDN using available tools to identify bottlenecks and optimize content delivery.
+- Optimize Cache Settings: Configure cache settings on your CDN to ensure that content is cached for an appropriate duration and that cache invalidation is handled correctly.
+- Use a Multi-CDN Strategy: Consider using multiple CDNs to improve redundancy, reduce latency, and optimize content delivery for users in different regions.
```

Comment:
I'm having problem to understand this strong recommendation. I guess Multi-CDN comes with its own complexity downsides. Could you provide a source which you used to provide this recommendation?